wolox-books/src/app/screens/BookDetail/components/book-detail/book-detail.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { BookService } from 'app/services/book.service';
+import { Book } from 'app/models/book';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-book-detail',
+  templateUrl: './book-detail.component.html',
+  styleUrls: ['./book-detail.component.scss']
+})
+export class BookDetailComponent implements OnInit {
+
+  book: Book = {
```

Comment:
I think there's no need for initializing this variable 🤔

Reply:
I made it to have like a "default" book in case the backend request fails, but of course that could be handled in many different ways. However I'll get rid of that 😅

Reply:
I better use conditions in the html to show "default things" if there's no book already created 👍